Mathlib/MeasureTheory/Integral/Bochner.lean

```diff
@@ -971,6 +977,14 @@ theorem norm_integral_le_lintegral_norm (f : α → G) :
   · simp [integral, hG]
 #align measure_theory.norm_integral_le_lintegral_norm MeasureTheory.norm_integral_le_lintegral_norm
 
+theorem nnnorm_integral_le_lintegral_nnnorm (f : α → G) : ‖∫ x, f x ∂μ‖₊ ≤ ∫⁻ x, ‖f x‖₊ ∂ μ := by
+  rw [integral_def]
+  split_ifs with hG hf
+  · calc _ ≤ (‖(Integrable.toL1 f hf)‖₊ : ℝ≥0∞) := by norm_cast; apply L1.nnnorm_integral_le
+      _ = _ := hf.nnnorm_toL1
+  · simp
+  · simp
+
 theorem ennnorm_integral_le_lintegral_ennnorm (f : α → G) :
     (‖∫ a, f a ∂μ‖₊ : ℝ≥0∞) ≤ ∫⁻ a, ‖f a‖₊ ∂μ := by
```

Comment:
Can this be golfed using the new result?

Reply:
Oops! It's the same result.